src/yo/index.ts

```diff
@@ -52,18 +53,16 @@ export async function generatorProject(addService) {
 
 	sub = subGenerator;
 
+	var beforeYo = getAllDirs(cwd);
+	var afterYo;
 	try {
-		const question: string = await yo.run(`${main}:${sub}`, cwd) as string;
-		if (!question) {
-			return;
-		}
-		const input = window.showInputBox({ prompt: question })
-		if (!input) {
-			return;
-		}
-		const argument = input;
-		await yo.run(`${main}:${sub} ${argument}`, cwd);
-
+		yo.run(`${main}:${sub}`, cwd).then(_p => {
+			afterYo = getAllDirs(cwd);
+			var newApp = _.difference(afterYo, beforeYo);
+			if (newApp.length > 0) {
```

Comment:
Should we check for newApp != null ?

Reply:
difference() will always return an array